src/poetry/console/commands/show.py

```diff
@@ -194,11 +194,11 @@ def _display_packages_information(
 
         from poetry.puzzle.solver import Solver
         from poetry.repositories.installed_repository import InstalledRepository
-        from poetry.repositories.pool import Pool
+        from poetry.repositories.repository_pool import RepositoryPool
         from poetry.utils.helpers import get_package_version_display_string
 
         locked_packages = locked_repository.packages
-        pool = Pool(ignore_repository_names=True)
+        pool = RepositoryPool(ignore_repository_names=True)
```

Comment:
Out of scope/for another time, but I am fascinated that we're using `RepositoryPool` here instead of just using `LockedRepository` directly.